### PR TITLE
restore vote-many command

### PR DESCRIPTION
**Pull request by nmarley**
I believe this got missed when moving from "budget" to "governance" code.

### CODE REVIEW

**Comment by UdjinM6** [comment]
utACK

File: src/rpcgovernance.cpp [hunk 1]
```diff
@@ -284,6 +285,100 @@ UniValue gobject(const UniValue& params, bool fHelp)
...
+    if(strCommand == "vote-many")
+    {
+        if(params.size() != 4)
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Correct usage is 'gobject vote-many <governance-hash> [funding|valid|delete] [yes|no|abstain]'");
+
+        uint256 hash;
+        std::string strVote;
+
+        hash = ParseHashV(params[1], "Object hash");
+        std::string strVoteAction = params[2].get_str();
```
Comment (tgflynn) [inline src/rpcgovernance.cpp]: I would suggest maintaining the established terminology of "signal" and "outcome".  It's already easy enough to confuse these without adding other terminology like "action".

Comment (nmarley) [inline src/rpcgovernance.cpp]: I did not add terminology. These are the exact same variable names used in vote-alias and vote-conf:

https://github.com/dashpay/dash/blob/v0.12.1.x/src/rpcgovernance.cpp#L221-L223
https://github.com/dashpay/dash/blob/v0.12.1.x/src/rpcgovernance.cpp#L299-L301

Comment (UdjinM6) [inline src/rpcgovernance.cpp]: I agree with you both :) Governance related modules imo desire some cleanup/refactoring. We can got either way but I think if everything else is ok I would rather merge this one and started another PR specifically for cleanup.

**Comment by tgflynn** [comment]
utACK

**Pull request merged**